tests/tests/datm_ocn_ice_mx100_cfsr

```diff
@@ -0,0 +1,23 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_COLD test
+#
+
+export TEST_DESCR="DATM-MOM6-CICE-CMEPS - 1 step cold start"
+
+export CNTL_DIR="datm_ocn_ice_mx100_cfsr"
+export CNTLMED_DIR="datm_ocn_ice_mx100_cfsr/RESTART"
+
+export LIST_FILES="RESTART/MOM.res.nc \
+                   RESTART/iced.2011-10-02-00000.nc \
+                   RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc \
+                   ocn_2011_10_01_21.nc"
+
+export_datm
+export DATM_SRC="CFSR"
+export FILENAME_BASE='cfsr.'
+export IATM=1760
+export JATM=880
+export FHMAX=24 # Note this needs to be at least 2x CPL_SLOW
```

Comment:
If FHMAX is the default, then we shouldn't re-set it again.

Reply:
FHMAX is removed.